### PR TITLE
Add azure-ai-anomalydetector to error code 5 ignored list

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Azure-ai-anomalydetector is autogenerated code so it doesn't have any tests. pytest returns error code 5 when no test is present and we need to ignore error code 5 for these packages.

### CODE REVIEW

File: eng/.docsettings.yml [hunk 1]
```diff
@@ -48,6 +48,7 @@ known_content_issues:
   - ['sdk/synapse/azure-synapse-spark/README.md', '#4554']
   - ['sdk/synapse/azure-synapse-artifacts/README.md', '#4554']
   - ['sdk/synapse/azure-synapse-nspkg/README.md', '#4554']
+  - ['sdk/anomalydetector/azure-ai-anomalydetector/README.md', '#4554']
```
Comment (weshaggard) [inline eng/.docsettings.yml:51]: FYI @mitchdenny added this in PR https://github.com/Azure/azure-sdk-for-python/pull/13141 at the bottom of the list so it is now in the list twice.

**Comment by mitchdenny** [comment]
Looks like this PR might have introduced some problems for this PR:

https://github.com/Azure/azure-sdk-for-python/pull/13135

**Comment by praveenkuttappan** [comment]
PR was to add anomalydetector ( new package) to override dependency and I believe this change is missing in current repo.

**Comment by mitchdenny** [comment]
Yep ... my bad, thanks for jumping onto it Praveen. I didn't notice that the pipeline run on the blocked PR was old, prior to  your fix going in, I had assumed that somehow your fix had introduced the problem.

**Pull request merged**